packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -225,6 +225,31 @@ In order to deploy a new version of this function:
 2. Re-deploy the stack (this will trigger a deployment).
 3. Monitor the CodeDeploy deployment as traffic shifts between the versions.
 
+#### Create a custom Deployment Config
+
+CodeDeploy for Lambda comes with built-in configurations for traffic shifting. If you want to specify your own strategy, you can do so with the CustomLambdaDeploymentConfig
+construct, letting you specify precisely how fast a new function version is deployed.
+
+```ts
+const config = new codedeploy.CustomLambdaDeploymentConfig(stack, 'CustomConfig', {
+  type: codedeploy.CustomLambdaDeploymentConfigType.CANARY,
+  interval: '1',
+  percentage: '5',
```

Comment:
These are out-of-date with the current code.

Reply:
Fixed